### PR TITLE
docs(snowflake): fix the syntax for passing schema

### PR DESCRIPTION
**Pull request by deepyaman**

### CODE REVIEW

File: docs/backends/snowflake.qmd [hunk 1]
```diff
@@ -119,7 +120,8 @@ APIs. All values of `authenticator` are supported.
 con = ibis.snowflake.connect(
     user="user",
     account="safpqpq-sq55555",
-    database="my_database/my_schema",
+    database="my_database",
```
Comment (deepyaman) [inline docs/backends/snowflake.qmd:123]:
```suggestion
    database="my_catalog/my_database",
```
Does it make sense to have something like this?

File: docs/backends/snowflake.qmd [hunk 2]
```diff
@@ -79,7 +79,8 @@ con = ibis.snowflake.connect(
     user="user",
     password="password",
     account="safpqpq-sq55555",
-    database="IBIS_TESTING/IBIS_TESTING",
+    database="IBIS_TESTING",
+    schema="IBIS_TESTING",
```
Comment (deepyaman) [inline docs/backends/snowflake.qmd:83]:
```suggestion
    database="my_database",
    schema="my_schema",
```

or 

```suggestion
    database="my_catalog/my_database",
    schema="my_schema",
```

may be more helpful?

File: docs/backends/snowflake.qmd (hunk 1 above)
Comment (cpcloud) [reply docs/backends/snowflake.qmd:123]: Hm, probably not. The idea is to do as little customization here so we can just dump kwargs to the underlying connector.

Comment (deepyaman) [reply docs/backends/snowflake.qmd:123]: Sorry, what I meant is, based on https://github.com/ibis-project/ibis/blob/9.3.0/ibis/backends/snowflake/tests/test_client.py#L411-L412, it seems like you can pass in the form `database="catalog/database"`. However, this isn't as clear from the examples, so I was wondering if it makes sense to document it as such.

Comment (cpcloud) [reply docs/backends/snowflake.qmd:123]: Sorry, I should clarify that I believe we removed that parsing functionality in 9.0. I don't think the comment is relevant anymore.

**Review by cpcloud — APPROVED** [review]

**Pull request merged**